app/views/about/hiring_rails_developer.html.haml

```diff
@@ -0,0 +1,207 @@
+        If so, this role might be perfect for you.
+
+      %h2.text-h2.mb-12 Summary
+      %p.text-p-xlarge.mb-16
+        Exercism is a not-for-profit organisation providing world-class programming education to anyone for free.
+        Our online platform is centred around self-directed learning and volunteer mentoring, and has nearly one million members.
+        Over the next 5 years, we want to become the defacto programming education platform that anyone, anywhere can use for free.
+        Through our work we aim to improve the global standard of programming and increase participation in tech from under-represented groups.
+        You can learn more about Exercism in our #{link_to '"About Exercism" pages', 'https://exercism.org/about', target: '_blank', rel: :noopener}.
+      %p.text-p-xlarge.mb-16
+        Exercism is a complex, highly-technical platform, with production code running in over 40 languages.
+        Our backend is centered around a Ruby on Rails backend (API, data and service layers) and a service oriented architecture (SOA) orchestrated using Ruby.
+        We run on AWS and use a variety of their different technologies.
+        We're looking for someone to take the lead in the continued development of Exercism's backend, implementing new features in the main Rails app, and extending our SOA with new functionality.
+      %p.text-p-xlarge.mb-40
+        We are building an exceptional team of ambitious, passionate people who love their work, excel at what they do, and have fun doing it.
+        The values of empathy, integrity and personal-growth are core to everything we do.
+        If that sounds exciting to be part of, and you feel you have the skills to help, we'd love for you to apply.
+
+      %h2.text-h2.mb-12 Responsibilities
+      %p.text-p-xlarge.mb-16
+        You'll be our primary backend developer, working daily on adding new features to Exercism's platform, and evolving and improving the ones that are already there.
+        This will involve working within our Rails codebase, our various microservices (and the piping between them) and working with our AWS account.
+      %p.text-p-xlarge.mb-16
+        As Exercism is a large and complex site, there is a huge variety in the things we work on day-to-day.
+        Here is an idea of some of the things you'll work on through the first 12 months:
+      %ul.text-p-xlarge.mb-20
+        %li Working with the front-end dev to build our new automated mentoring management system.
+        %li Moving from our current mail provider to SES, writing a webhooks API to store underlying deliverability data (and saving us 90% costs on email!)
+        %li Developing out a prototype for our new Teams platform.
+        %li Adding a variety of action/reaction mechanisms to improve retention.
+        %li Working through many of the <a href="https://github.com/exercism/exercism/issues">open issues</a> that our community have opened.
```

Comment:
Shouldn't this use the haml interpolated link?

Reply:
Yeah, probably!